crates/control/src/led_status.rs

```diff
@@ -147,9 +148,17 @@ impl LedStatus {
             last_ball_data_top_too_old,
             last_ball_data_bottom_too_old,
         );
+
+        let ears = if context.filtered_whistle.is_detected {
```

Comment:
How long is `is_detected` true? If only one cycle, is this noticeable?

Reply:
When testing this the ears just flashed for a couple cycles for the shortest whistle burst I could create but were lit up continuously for blowing the whistle for longer.